typescript/infra/src/config/environment.ts

```diff
@@ -24,19 +24,19 @@ export type EnvironmentChain<E extends DeployEnvironment> = Extract<
   ChainName
 >;
 
-export type CoreEnvironmentConfig<Chain extends ChainName> = {
+export type CoreEnvironmentConfig = {
   environment: DeployEnvironment;
-  transactionConfigs: EnvironmentConfig<Chain>;
+  transactionConfigs: ChainMap<ChainMetadata>;
```

Comment:
Should we maybe rename these from transactionConfigs to chainMetadatas?

Reply:
Love it